doc/usermacros.rst

```diff
@@ -1,13 +1,13 @@
 User macro reference
 ====================
 
-.. cmake:macro:: add_message_files(DIRECTORY dir FILES file1 [file2...] [PACKAGE pkgname] [NOINSTALL])
+.. cmake:macro:: add_message_files(DIRECTORY dir FILES file1 [file2...] [BASE_DIR base_dir] [NOINSTALL])
 
    :param DIRECTORY: Directory containing messages.  May be absolute or
-     relative to ``CMAKE_CURRENT_SOURCE_DIR``.
+     relative to ``CMAKE_CURRENT_SOURCE_DIR``.  The name will also be used as the install folder. Default is `msg`
```

Comment:
If you want to mention the install location it should be the "full" path: `share/<pkgname>/<directory>`